posthog/models/organization.py

```diff
@@ -9,6 +9,7 @@
 from django.db.models.query_utils import Q
 from django.dispatch import receiver
 from django.utils import timezone
+from jsonschema import ValidationError
```

Comment:
This might be the wrong import. Probably meant to use Django ValidationError.

Reply:
Oh good catch!